tests/Support/SupportTestingQueueFakeTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Illuminate\Tests\Support;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Foundation\Application;
+use Illuminate\Support\Testing\Fakes\QueueFake;
+use PHPUnit\Framework\ExpectationFailedException;
+
+class QueueFakeTest extends TestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->fake = new QueueFake(new Application);
+        $this->job = new JobStub;
+    }
+
+    public function testAssertPushed()
+    {
+        try {
+            $this->fake->assertPushed(JobStub::class);
+        } catch (ExpectationFailedException $exception) {
+            $this->assertEquals('The expected [Illuminate\Tests\Support\JobStub] job was not pushed.
```

Comment:
We use `@expectedException` and `@expectedExceptionMessage` docblocks for all other tests.